### PR TITLE
Login page's UI made similar to register page's UI

### PR DESCRIPTION
**Pull request by manasivaraiya**
## Related Issue or bug
  - Info about Issue or bug

Fixes: #160 

#### Describe the changes you've made
A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 **original screenshot**  | <b>updated screenshot </b> |

### CODE REVIEW

File: main_app/templates/main_app/login.html [hunk 1]
```diff
@@ -58,22 +130,18 @@ <h1 class="display-4" style="text-align: center; color: black; font-family: 'Ban
                         Registered
```
Comment (sakshikhachane) [inline main_app/templates/main_app/login.html:130]: - [ ] Change `WELCOME` to `WELCOME TO RESCUE` and apply CSS to it similar to `register` page.
- [ ] Change the image of `login` page to the image present on the `register` page.

File: main_app/templates/main_app/login.html [hunk 2]
```diff
@@ -58,22 +130,18 @@ <h1 class="display-4" style="text-align: center; color: black; font-family: 'Ban
...
                         style="margin-left: 30%; height: 35%; width: 35%;">
                     {% csrf_token %}
                     {{form.as_p}}
+                    
                     <br>
-                    <div style="display: flex; align-items: center; justify-content: center;">
-                        <button style="border-radius: 25px;" class="btn btn-success" type="submit"><b>Sign
-                                In</b></button>
+                    
+                    <div class="login">
```
Comment (sakshikhachane) [inline main_app/templates/main_app/login.html:141]: Please fix the space above `WELCOME` for the devices with smaller widths and heights.
image

**Review by sakshikhachane — CHANGES_REQUESTED** [review]
@manasivaraiya Please do the requested changes.

**Comment by sakshikhachane** [comment]
@manasivaraiya Any updates?

**Comment by manasivaraiya** [comment]
In my repo I do not find any such space above welcome on login page! Could you please check once again and let me know?
I have committed the 2 changes proposed above .

**Comment by manasivaraiya** [comment]
> @manasivaraiya Any updates?

really sorry for the late reply.

**Review by sakshikhachane — APPROVED** [review]

**Comment by sakshikhachane** [comment]
@manasivaraiya Please resolve the conflicts.

**Comment by mansijain980** [comment]
@sakshikhachane can i proceed further?

**Comment by swapnilsparsh** [comment]
@manasivaraiya Any updates ?

**Pull request closed without merging**